android/services/mongodb-remote-perftests/build.gradle

```diff
@@ -31,11 +31,35 @@ android {
         if (!properties.getProperty("test.stitch.androidPerfStitchAPIKey", "").isEmpty()) {
             testInstrumentationRunnerArgument "test.stitch.androidPerfStitchAPIKey", properties.getProperty("test.stitch.androidPerfStitchAPIKey", "")
         }
-        if (!properties.getProperty("test.stitch.androidPerfStitchHostname", "").isEmpty()) {
-            testInstrumentationRunnerArgument "test.stitch.androidPerfStitchHostname", properties.getProperty("test.stitch.androidPerfStitchHostname", "")
+        if (!properties.getProperty("test.stitch.perf.stitchHost", "").isEmpty()) {
```

Comment:
[nit] can we make an array of strings and do these in a for loop so it's a bit cleaner?